plugin.picture.googlephotos/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,313 @@
+# Kodi Media Center language file
+# Addon Name: Google Photos 
+# Addon id: plugin.picture.googlephotos
+# Addon Provider: Pranjal Singhal
+msgid ""        
+msgstr ""
+
+#settings screen settings.xml
+#setting category
+msgctxt "#30100"
+msgid "Sign-in Server"
+msgstr ""
+
+msgctxt "#30101"
+msgid "Base URL"
+msgstr ""
+
+msgctxt "#30102"
+msgid "Device Code Subroute"
+msgstr ""
+
+msgctxt "#30103"
+msgid "Access Token Subroute"
+msgstr ""
+
+msgctxt "#30104"
+msgid "Refresh Token Subroute"
+msgstr ""
+
+msgctxt "#30105"
+msgid "https://photoslibrary.googleapis.com/v1"
```

Comment:
Not used, and definitely something we don't want translated into something malicious.

Reply:
Removed